skimage/__init__.py

```diff
@@ -117,68 +86,42 @@ def _raise_build_error(e):
     import sys
```

Comment:
`__SKIMAGE_SETUP__` is no longer injected anywhere AFAICT, so we can get rid of this too.

Reply:
Good catch! Removed in https://github.com/scikit-image/scikit-image/pull/6892/commits/f4e3218205e4059dd67fb6142f69900d21bef856.